lua/refactoring/utils.lua

```diff
@@ -194,11 +217,12 @@ end
 ---@param node TSNode
 function M.region_above_node(node)
     local scope_region = Region:from_node(node)
-    local lsp_range = scope_region:to_lsp_range()
+    -- TODO (TheLeoP): should I create a special case for this?
```

Comment:
Yeah I think it's best just to figure out whether to call replace or insert. Best not to mess with behaviour of util functions unless we need to.